src/components/map/mapMarker/index.jsx

```diff
@@ -112,9 +112,6 @@ const getMarkerSVG = (color, strokeColor = '#000000') => {
   return `
     <svg fill="${color}" width="30px" height="30px" viewBox="-51.2 -51.2 614.40 614.40" xmlns="http://www.w3.org/2000/svg">
       <g id="SVGRepo_bgCarrier" stroke-width="0"></g>
-      <g id="SVGRepo_tracerCarrier" stroke-linecap="round" stroke-linejoin="round" stroke="${strokeColor}" stroke-width="10.24">
```

Comment:
Try to see if we can still put this by using a zero opacity color. Stroke could be useful in other usecases.